workers/unity/Assets/Playground/Scripts/Worker/AbstractMobileClientWorker.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Improbable.Gdk.Core;
+using Improbable.Gdk.Mobile;
+using Playground;
+using UnityEngine;
+using UnityEngine.UI;
+
+public abstract class AbstractMobileClientWorker : MobileWorkerConnector
+{
+    public GameObject ConnectionPanel;
+    public GameObject Level;
+
+    private GameObject levelInstance;
+    private bool connected;
+    private InputField input;
+    private Button button;
+    private Text errorMessage;
+
+    public void Awake()
+    {
+        input = ConnectionPanel.transform.Find("ConnectInput").GetComponent<InputField>();
+        button = ConnectionPanel.transform.Find("ConnectButton").GetComponent<Button>();
+        errorMessage = ConnectionPanel.transform.Find("ConnectionError").GetComponent<Text>();
```

Comment:
Can't these be linked in the editor?

Reply:
They can. Is it a better practice to link both the parent object and its children as opposed to just linking the parent and getting the children from there? I was not aware.

Reply:
better to link as it means that Unity doesn't need to search for stuff during runtime

Reply:
More explicit / less brittle